src/mlpack/tests/layer_names_test.cpp

```diff
@@ -63,69 +60,69 @@ BOOST_AUTO_TEST_CASE(LayerNameVisitorTest)
   // Bilinear interpolation is not yet supported by the string converter.
   LayerTypes<> unsupportedLayer = new BilinearInterpolation<>();
 
-  BOOST_REQUIRE(boost::apply_visitor(LayerNameVisitor(),
+  REQUIRE(boost::apply_visitor(LayerNameVisitor(),
                       atrousConvolution) == "atrousconvolution");
```

Comment:
I know this is tedious, but maybe you can wrap these second lines so they're all doubly-indented from the first?
```suggestion
      atrousConvolution) == "atrousconvolution");
```